src/app/issues/[id]/author/page.tsx

```diff
@@ -0,0 +1,23 @@
+import AuthorViewpointCard from "@/components/AuthorViewpoint/AuthorViewpointCard";
+import { Metadata } from "next";
+
+type AuthorViewPointProps = {
+    params: Promise<{ id: string }>;
+};
+
+export const metadata: Metadata = {
+    title: "CommonGround - 撰寫觀點",
+    keywords: "社會時事, 觀點, 理性討論, 撰寫觀點",
+};
+
+export default async function AuthorViewPoint({
+    params,
+}: AuthorViewPointProps) {
+    const id = (await params).id;
+
+    return (
+        <main className="mx-auto my-8 w-full max-w-7xl">
+            <AuthorViewpointCard issueId={id} />
```

Comment:
I think you probably could make this whole page `use client` and move the logic from AuthorViewpointCard to here

Reply:
If the whole page is `use client`: 
First, I can't export metadata. I need to add the header in the page to overwrite the default HTML header.
Second, client page does not yet support `await` method. Since the page params is a Promise object in Next js 15, some work around like react router or React.use() is needed. 
For the reason above, I personally don't think making the whole page `use client` will let the structure look better, instead, it suffers the risk of breaking the consistency of code in our pages.

Reply:
I think in principle making the page client side rendered makes more sense as the page serves only as an editor. On the Head front, the [`<Head>`](https://nextjs.org/docs/pages/api-reference/components/head) component in Next.JS should suffice and shouldn't be too hard to setup. Furthermore, getting the dynamic URL client side is also applicable using the [`useRouter`](https://nextjs.org/docs/pages/api-reference/components/head) hook (don't worry, its compatible with the app router). Thus, I think it still makes sense to render the page client side.

Reply:
Ok, I think you persuaded me. Then I suppose that we should do the same to `Onboarding page` too.

Reply:
unfortunately `<Head>` dose not work for app router